float64 tcp 
float64 ori
float64 droopAngle
geometry_msgs/Pose pose
